src/Tasks.php

```diff
@@ -1036,27 +1042,38 @@ private function downloadPantheonBackup($env) {
   }
 
   /**
-   * Imports a local database.
+   * Imports a local database using mysql rather than zcat.
    *
    * @return bool
    *   True if import succeeded.
    */
   public function importLocal() {
+    if (!$this->taskExec('which pv')->run()->wasSuccessful()) {
+      $this->yell("This project's database can take up to an hour to import. So that you can be informed of its progress, please install the pv tool with 'brew install pv', then run your command again.");
+      return FALSE;
+    }
+
     $project_properties = $this->getProjectProperties();
 
+    $web_root = $project_properties['web_root'];
+    $sql_db = $project_properties['db-name'];
+    $sql_user = $project_properties['db-user'];
+    $sql_pass = $project_properties['db-pass'];
     // Empty out the old database so deleted tables don't stick around.
-    $this->taskExec('drush sql:drop -y')->dir($project_properties['web_root'])->run();
-
-    $drush_commands    = [
-      'drush_import_database' => 'zcat < ../vendor/database.sql.gz | drush @self sqlc # Importing local copy of db.'
+    $this->taskExec('drush sql:drop -y')->dir($web_root)->run();
+    if (file_exists('vendor/database.sql.gz')) {
+      $this->taskExec('gunzip ../vendor/database.sql.gz')->dir($web_root)->run();
```

Comment:
Let's move this to be part of `downloadPantheonBackup` instead so the amount of time there is a `vendor/database.sql.gz` file is minimal, and then we can ignore its existence everywhere else.

Reply:
Actually... How much more space does the unzipped version take up? Maybe it's worth it to keep zcat just to save file space on our computers?

Reply:
Good call, it's a LOT. But gunzip doesn't leave the old version behind by default. maybe there's an option!

Reply:
`-k` it looks like

Reply:
oh I see you just skip it and have zcat directly go into the pipe!